src/l500/l500-private.h

```diff
@@ -261,21 +261,6 @@ namespace librealsense
             { DFU_error,                    "DFU error" },
         };
 
-        template<class T>
-        const T* check_calib(const std::vector<uint8_t>& raw_data)
```

Comment:
I think it should be reverted and adjusted to calb table template as needed

Reply:
As discussed separately, there was a miss understanding.
The current implementation is OK  for the given table structure